peerdroid/src/main/java/rdx/works/peerdroid/data/PeerdroidLink.kt

```diff
@@ -45,9 +51,22 @@ interface PeerdroidLink {
 
     suspend fun sendMessage(
         connectionId: String,
-        message: ConnectorExtensionExchangeInteraction
+        message: LinkClientExchangeInteraction
     ): Result<Unit>
 
+    @Serializable
+    data class LinkClientExchangeInteraction(
```

Comment:
are there any other ExchangeInteractions that are planned or we do not support yet, but CE supports? Using discriminator suggests polymorphic serialization

Reply:
There are no other types now, but there might be in the future, I guess the discriminator here is more for consistency when interacting with CE